src/shared/Atlassian.Bitbucket/BitbucketHostProvider.cs

```diff
@@ -43,6 +44,11 @@ public bool IsSupported(InputArguments input)
                 return false;
             }
 
+            if (input.WwwAuth.Any(x => x.Contains("realm=\"Atlassian Bitbucket\"")))
```

Comment:
I wonder if we want to be a little more broad in string matching and ignore the case?

Reply:
Good idea, done.